src/smc-webapp/project_store.coffee

```diff
@@ -1200,15 +1200,17 @@ class ProjectActions extends Actions
                 if target[target.length-1] == '/' or full_path == ''
                     @open_directory(parent_path)
                 else
-                    @fetch_directory_listing(parent_path)
+                    @fetch_directory_listing(parent_path, true, true)
                     @get_store().wait
-                        until   : (s) => s.directory_listings.get(parent_path)
+                        until   : (s) =>
+                            listing = s.directory_listings.get(parent_path)
+                            return listing?.find (val) => val.get('name') == last
                         timeout : 3
-                        cb      : (err, listing) =>
+                        cb      : (err, item) =>
                             if err
-                                alert_message(type:'error', message:'Failed to open link')
+                                console.log err
```

Comment:
Yes, because right now I can't seem to reproduce #1359 anywhere except on live and even then only rarely. In this case, I want to find out what the error is first because the error might not even matter to the user. 

I think opening the parent directory is fine though.